README.md

```diff
@@ -1,4 +1,177 @@
-### Running notebooks
+# openEO process graph to evalscript converter
+This repository contains a library for converting openEO process graphs to [Sentinel Hub evalscripts](https://docs.sentinel-hub.com/api/latest/evalscript/v3/).
```

Comment:
Maybe also link tho the process graph API documentation here: [openEO process graphs](https://api.openeo.org/#section/Processes/Process-Graphs)

Reply:
Linked.